fern/docs/pages/manual/nlsql.mdx

```diff
@@ -0,0 +1,30 @@
+## MySQL Configuration
+Text-to-SQL querying has been tested for [MySQL](https://www.mysql.com/) in PrivateGPT.
+
+To use the text to SQL feature, set the following environment variables to allow connecting to your MySQL Server:
+
+```yaml
+sqldatabase:
+  dialect: mysql
+  driver: pymysql
+  host: <db_hostname_here>
+  user: <db_username_here>
+  password: <db_password_here>
+  database: <database_name_here>
+  tables: <list_of_table_names_here>
+```
+
+You will also need to install `pymysql`.
+
+`poetry add pymysql`
+
+## Other Databases
+To get started with exploring the use of other databases, set the `sqldatabase.dialect` and `sqldatabase.driver` properties in the `settings.yaml` file.
+
+```yaml
+sqldatabase:
+  dialect: <dialect_here>
+  driver: <driver_here>
+```
+
+Refer to [SQLAlchemy Engine Configuration](https://docs.sqlalchemy.org/en/20/core/engines.html) for constructing the DB connection string.
```

Comment:
That would be great if that section could be a bit more user friendly - for example, putting additional links to SQLAlchemy reference (for example, the list of `dialect` https://docs.sqlalchemy.org/en/20/dialects/), and at least one additional link (to postgres or sqlite for example, which are very common).